app/javascript/packages/form-steps/form-steps-continue-button.tsx

```diff
@@ -12,12 +12,12 @@ interface FormStepsContinueButtonProps {
 
 function FormStepsContinueButton({ className }: FormStepsContinueButtonProps) {
   const { t } = useI18n();
-  const { isLastStep } = useContext(FormStepsContext);
+  const { isLastStep, toNextStep } = useContext(FormStepsContext);
 
   const classes = ['display-block', 'margin-y-5', className].filter(Boolean).join(' ');
 
   return (
-    <Button type="submit" isBig isWide className={classes}>
+    <Button onClick={toNextStep} isBig isWide className={classes}>
```

Comment:
This change is necessary because the personal key confirmation modal content is rendered outside the `<form>` element, due to the `createPortal` usage in `<FullScreen />`. While [React does support event bubbling through portals](https://reactjs.org/docs/portals.html#event-bubbling-through-portals), the `submit` event is never fired by the browser, since the button is technically not within a form.